### ISSUE TITLE
Toggle Columns Throw DOMException: Failed to execute 'replaceState' on 'History': #<Object> could not be cloned

### ISSUE THREAD

**Issue by alhawaj98** [protonemedia/laravel-splade#140]
### Versions
- `protonemedia/laravel-splade` : 1.0.4
- `laravel/framework` : 9.19

### Describe the problem:
I'm using Splade Table component. In the event of toggle table column `this.$splade.replaceUrlOfCurrentPage(url)` throw the following exception:
`Uncaught DOMException: Failed to execute 'replaceState' on 'History': [object Array] could not be cloned.`

![image](https://user-images.githubusercontent.com/31033498/206647341-c8764525-d0fc-40e4-af1a-29e008a6b3f6.png)

**Comment by pascalbaljet** [comment]
Can you share the code of the controller and the Blade template? Thanks!

**Comment by alhawaj98** [comment]
I have a simple route & blade: 

Route:

```
Route::get('/', function (){
return view("pages.tenant.home", [ 'users' => Users::class]);})->name('tenant.home');
}
```

Blade:

```
<x-layout.main.layout :headerMenus="$headerMenus ?? []">
            <x-splade-table :for="$users" :striped="true" />
</x-layout.main.layout>
```
Users Class :

```
    public function configure(SpladeTable $table)
    {
        $table
            ->withGlobalSearch(columns: ['id', "login"])
            ->column('id', sortable: true)
            ->column('login', sortable: true)
            ->column('active', sortable: true)
            ->searchInput("id")
            ->searchInput("login")
            ->searchInput("active")
            ->export();
    }

```

I'm working on a new project where each client has his own database & subdomain by using [Tenancy for Laravel Pacakge](https://tenancyforlaravel.com/). I'm not sure if this issue related to `Tenancy  package` or not. However I don't see how

 `replace` function is working fine
```          
 this.$splade.replace(url)
```

but `replaceUrlOfCurrentPage` throw an exception:
```
  return this.$splade.replaceUrlOfCurrentPage(url);
```

**Comment by guiofranca** [comment]
Hello!

I'm having the same issue with the straight forward [sample in the docs](https://splade.dev/docs/table-overview). Whenever I toggle the visibility on any column, the url is not updated and there's the reported error on dev console.

- `Laravel`: 9.45.1
- `protonemedia/laravel-splade-breeze`: 2.1.0

**Comment by LordMcAwesome** [comment]
For me this exception is also thrown in a Vue Component where I import "Splade" as such:
```
import {inject} from "vue";
const Splade = inject("$splade");
```

and then try to append a query String to the URL:
```
var queryParams = new URLSearchParams(window.location.search);
...
Splade.replaceUrlOfCurrentPage("?" + queryParams.toString());
```

**Comment by pascalbaljet** [comment]
Thanks for sharing, everybody! I can reproduce the problem so I'll dive into it :)

**Comment by pascalbaljet** [comment]
Fixed in v1.1.4. Make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).

**Comment by LordMcAwesome** [comment]
Works as well like a charm. Thank you @pascalbaljet :)
